apps/dashboard/src/pages/group.tsx

```diff
@@ -479,6 +508,28 @@ ${memberIds.join("\n")}
                             )}
                         </Box>
                     )}
+                    {_group.type === "off-chain" && isGroupAdmin && (
+                        <Box
+                            bgColor="balticSea.50"
+                            p="25px 30px 25px 30px"
+                            borderRadius="8px"
+                        >
+                        <Text fontSize="20px">Download group</Text>
+                        <Flex align="center" mt="2">
+                            <Text my="10px" fontWeight="400">
+                                Get the group's data in JSON format.
```

Comment:
```suggestion
                                Get the group&apos;s data in JSON format.
```

This should fix the lint error you see when running `yan lint`

Reply:
Thank you for the check. I've made the requested changes:

- Ran `yarn prettier:write` to format the code
- Executed `yarn lint` and fixed the reported issues